src/main/java/org/springframework/samples/petclinic/math/MathController.java

```diff
@@ -14,6 +19,13 @@ public MathController(MathService mathService) {
 		this.mathService = mathService;
 	}
 
+	@Operation( summary = "Add 2 numbers.",
+		description = "Adds 2 numbers that can be summed.",
+		tags = { "Math" })
+	@ApiResponses(value = {
+		@ApiResponse(responseCode = "200", description = "successful add", content = @Content(examples = @ExampleObject("0"))),
```

Comment:
Not an efficient way but it is to show that any string object can provided in place of "0" to customly define default return data structure.